### ISSUE TITLE
Trouble plotting ODE solutions due to `lambdify` function

### ISSUE THREAD

**Issue by 00krishna** [JuliaPy/SymPy.jl#298]
This issue follows on from a Discourse Post that raised the issue. 

https://discourse.julialang.org/t/plotting-a-differential-equation-using-sympy-jl-and-plots-jl/28876/4

The point is that SymPy can plot symbolic equations, but for some reason it has trouble plotting ODE solutions, which are also symbolic equations. 

Here is a simple example:
```
using SymPy
@vars t
F = SymFunction("F")
diffeq = F'(t) - 3*F(t)
res = dsolve(diffeq, F(t), ics=(F, 0, 2))

using Plots
plot(res, -1, 5) 
```
The resulting message is:
```
ERROR: MethodError: no method matching ##465(::Float64)
Closest candidates are:
  ##465(::Any, ::Any) at none:0
```

According to j_verzani on Discourse, I could use something like this:
`plot(rhs(res), 0, 2 )` to plot the function, but this code is a little confusing because it is inconsistent with the way that other symbolic equations are plotted. In most cases a symbolic equation just has a single right hand side of the equation, but ODE solutions have both a right and left hand side--hence the explicit call to `rhs()`. 

As j_verzani suggested, a simple fix would be to implement the `lambdify()` function for expressions like an ODE solution. 

Let me know if you have any questions or require any additional information. Thanks.

**Comment by 00krishna** [comment]
Hello @jverzani thanks for submitting the fix for this issue. I was just wondering when the change will be released? Will this change be included if I update the current SymPy package v1.0.6, or do I need to wait till version 1.0.7 is released, etc? The workaround using `plot(rhs(res), 0, 2 )` is working just fine, but just wanted to know for the future. Thanks.

**Comment by jverzani** [comment]
Version 1.0.7 will include this change. It should be available within a day or so if things go as usual.
